windows/datashare.nsi

```diff
@@ -127,24 +127,24 @@ FunctionEnd
 
 Function InstallOpenJre64
     #Java lib test
-    nsExec::ExecToStack "java -version"
+    nsExec::ExecToStack '"$SYSDIR\cmd.exe" /c where -f java | findstr -R "[jdk|jre]-11" | cmd /e /v /q /c"set/p.=&&echo(^!.^!"'
```

Comment:
List installed Java versions | Grep version 11 | Keep the first element ( we don't mind the minor version)

Reply:
Thanks for the translation